integrations/mixpanel/lib/index.js

```diff
@@ -37,6 +37,7 @@ var Mixpanel = (module.exports = integration('Mixpanel')
   .option('trackAllPages', false)
   .option('trackNamedPages', false)
   .option('trackCategorizedPages', false)
+  .option('groupIdentifierTraits', [])
```

Comment:
Is `groupIdentifierTraits` a new setting? If so, we should should either run a DCS backfill or ensure we don't call any array methods on it until we check that it's not null/undefined and check the type.

Reply:
Is not a new setting.